README.md

```diff
@@ -41,17 +41,14 @@ Test Matrix
 ------------
 | Andrid Studio Version | cmake in SDK| NDK      | Result  |
 |-----------------------|-------------|----------|---------|
-| 2.2.0-beta1           | 3.6.3133135 | NDK-r12  | Yes     |
-| 2.2.0-beta1           | 3.6.3133135 | r13-beta1| NO      |
+| 3.0.0                 | 3.6.4111459 | NDK-r16  | Yes     |
 
 Known Issue:
- - r13-beta layer source needed to be updated
- - Studio Beta cmake 3.6.3133135 having issues with r13-beta
 
 Getting Started
 ---------------
 1. [Download Android Studio](http://tools.android.com/download/studio/canary)
-1. [Download Android NDK-R12+](https://github.com/android-ndk/ndk/wiki)
+1. [Download Android NDK-r16+](https://github.com/android-ndk/ndk/wiki)
 1. [Download source code](http://www.github.com/googlesamples/android-vulkan-tutorials)
 1. Launch Android Studio.
 1. "Import Project" of the interested tutorial project to Android Studio
```

Comment:
Perhaps something like this would be more clear?

- From the Welcome screen, select "Import Project" (or, if you already have a project open, select __File > New > Import Project__).
- Navigate to the sample you downloaded and click __OK__

Reply:
done